pom.xml

```diff
@@ -56,6 +56,7 @@
         <dep.testng.version>6.10</dep.testng.version>
         <dep.nifty.version>0.15.1</dep.nifty.version>
         <dep.swift.version>0.15.2</dep.swift.version>
+        <dep.kafka-clients.version>0.10.2.1</dep.kafka-clients.version>
```

Comment:
Don't add a property for a version that is only used in one place. We use properties for related artifacts that need to be versioned together.